config/initializers/devise.rb

```diff
@@ -272,11 +272,13 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-  google_client_id = Rails.application.credentials.google_oauth2[:client_id]
-  google_client_secret = Rails.application.credentials.google_oauth2[:client_secret]
-  config.omniauth :google_oauth2, google_client_id, google_client_secret,
-                  scope: 'userinfo.email,userinfo.profile', skip_jwt: true
-
+  # google_client_id = Rails.application.credentials.google_oauth2[:client_id]
+  # google_client_secret = Rails.application.credentials.google_oauth2[:client_secret]
+  # config.omniauth :google_oauth2, google_client_id, google_client_secret,
+  #                 scope: 'userinfo.email,userinfo.profile', skip_jwt: true
+  GOOGLE_CLIENT_ID = '936253024035-pmkppsgge4psmcpserbdk8e05o7pd3ub.apps.googleusercontent.com'
```

Comment:
One more thing, it's a big security issue to put store credentials in the public files

Reply:
＋

Reply:
Fixed